app/views/admin/enterprises/_actions.html.haml

```diff
@@ -5,6 +5,8 @@
   = link_to_delete_enterprise enterprise
   %br/
 
+<div class="admin-enterprises-index__admin-actions-divider"></div>
+
```

Comment:
Would a horizontal rule (hr) element not be better here? Or is there a reason to use a div with height 1px?

Also, we don't need all these braces and closing bracket in haml. `%div.classname` is fine, or as a div is the most common element you can skip it all together and just use `.classname` as a shorthand.